docs/development/new-packages.md

```diff
@@ -17,6 +17,18 @@ Most pure Python packages can be installed directly from PyPI with
 {func}`micropip.install` if they have a pure Python wheel. Check if this is the
 case by trying `micropip.install("package-name")`.
 
+Due to [current limitations of WebAssembly](https://pyodide.org/en/stable/usage/wasm-constraints.html),
+packages that make heavy use of multi-threading and multi-processing **may not** work
+without explicit support from maintainers. A workaround could be to avoid creating multiple
+threads when on WASM using a check like:
```

Comment:
Maybe:
```suggestion
Because Pyodide [does not support threading or multiprocessing](https://pyodide.org/en/stable/usage/wasm-constraints.html),
packages that use threading or multiprocessing will not work without a patch to disable it. For example,
```